bears/js/JSONFormatBear.py

```diff
@@ -37,10 +38,16 @@ def run(self, filename, file,
             json_content = json.loads(''.join(file),
                                       object_pairs_hook=OrderedDict)
         except JSONDecodeError as err:
+            err_content = str(err).split(':')
+            err_msg = err_content[0]+'.'
+            position = findall(r'\d+', err_content[1])
```

Comment:
I think you should make a single regex and use capture groups accordingly, so your parsing just requires a single match:
```
m = match(r'(.*): line (\d+) column (\d+)'
err_msg = m.group(1) + '.'
...
    line=int(m.group(1))
    column=int(m.group(2))
```